manifests/d/DavidRios/Remindr/2.0.2/DavidRios.Remindr.locale.en-US.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: DavidRios.Remindr
+PackageVersion: 2.0.2
+PackageLocale: en-US
+Publisher: David Rios
+PublisherUrl: https://github.com/MrDavidRios/remindr_releases
+PublisherSupportUrl: https://github.com/MrDavidRios/remindr_releases/issues
+Author: David Rios
+PackageName: Remindr
+PackageUrl: https://mrdavidrios.github.io/remindr
+License: MIT

```

Comment:
Is there a LicenseUrl for this? I can't seem to find license related info on the GitHub / PackageUrl. If no info about license is availbale, this should be set to `Proprietary`